inst/schema/PlottingMetadataResponse.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
```

Comment:
it looks like `{survey,anc,output}_chloropleth` refer to the same type, and so you could simplify this

Reply:
They do, though this is one of the things I expect might change after talking to Jeff. I've simplified it for now though as you suggest.